ssclj/rpm/src/main/systemd/ssclj.service

```diff
@@ -1,13 +1,22 @@
 [Unit]
 Description=SlipStream API
-After=syslog.target network.target
+After=syslog.target
+After=network.target
```

Comment:
I have another question regarding `create-daily-usage.sh`
should `-cp /opt/slipstream/ssclj/etc` be updated?